package.json

```diff
@@ -29,6 +29,7 @@
     "coverageDirectory": "./coverage"
   },
   "dependencies": {
+    "form-data": "^4.0.0",
```

Comment:
It seems like we only use exact versions for packages here. (Maybe we should specify that in an .npmrc)

Reply:
Yes, I think it would be good to specify that in an `.npmrc`.
But I'm not sure if we should add it in this PR or in a separate one.